registration_2019/dayof.py

```diff
@@ -62,22 +81,26 @@ class SignInEndpoint(Resource):
     def __init__(self):
 
         self.parser.add_argument('user_id',            type=strn, required=True)
-        self.parser.add_argument('badge_data',         type=strn, required=True)
+        self.parser.add_argument('badge_data',         type=badge_data, required=True)
 
     @auth
     def post(self):
         args = self.parser.parse_args()
         return sign_in(args['user_id'], args['badge_data'])
 
     def get(self):
```

Comment:
Should be auth'd

Reply:
Not sure what you're referring to, if its the GET method, I don't think it needs to be as it only shows counts. The Post method is authed.